### PR TITLE
Fix #134 #136

### LINKED ISSUE #134: Don't re-create a secure queue with POST

### PR DESCRIPTION
**Pull request by crbrox**

### CODE REVIEW

File: lib/dataHelper.js [hunk 1]
```diff
@@ -252,9 +263,8 @@ exports.setExpirationDate = setExpirationDate;
  * @param callback
  */
 exports.setKey = setKey;
```
Comment (aitormagan) [inline lib/dataHelper.js:266]: This function is not used now. Can it be deleted?

File: lib/dataHelper.js [hunk 2]
```diff
@@ -28,14 +28,25 @@ var log = require('PDITCLogger');
 var logger = log.newLogger();
 logger.prefix = path.basename(module.filename, '.js');
 
+var createKey = function(db, id, value, callback) {
+    db.setnx(id, value, function onSet(err,val) {
+        if (err) {
+            logger.warning(err);
+        }
+        else if(val === 0) {
+            err = new Error(id + " exists");
```
Comment (aitormagan) [inline lib/dataHelper.js]: ID includes app prefix, so the error returned to the user is `"Error: SEC:q1 exists"` when it should be `"Error: q1 exists"`

**Comment by aitormagan** [comment]
Status code returned when a queue previously exists is 500 but it should be 403.
Error message is returned within an array instead of an object.

**Comment by aitormagan** [comment]
Before merge, queueState test must be updated (I've already updated it, but I will push it when errors are fixed).

**Comment by aitormagan** [comment]
Can `setKey` function be deleted from `dataHelper.js`?

**Comment by mrutid** [comment]
Sure it can... but should? 

Unused functions will be removed during refactor phase (next sprint).

**Pull request merged**